src/backend/efiling-hub-api/src/test/java/ca/bc/gov/open/jag/efilinghubapi/submission/submissionApiDelegateImpl/UploadSubmissionDocumentsTest.java

```diff
@@ -0,0 +1,25 @@
+package ca.bc.gov.open.jag.efilinghubapi.submission.submissionApiDelegateImpl;
+
+import ca.bc.gov.open.jag.efilinghub.api.model.UploadSubmissionDocumentsResponse;
+import ca.bc.gov.open.jag.efilinghubapi.submission.SubmissionApiDelegateImpl;
+import org.junit.jupiter.api.*;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+
+@TestInstance(TestInstance.Lifecycle.PER_CLASS)
+@DisplayName("SubmissionApiDelegateImpl test suite")
+public class UploadSubmissionDocumentsTest {
+    private SubmissionApiDelegateImpl sut;
+    @BeforeAll
+    public void setUp() {
+        sut = new SubmissionApiDelegateImpl();
+    }
+    @Test
+    @DisplayName("200: Valid request should add documents")
```

Comment:
should there be a test case for non-valid request?

Reply:
This is just the basic implementation it only return 200